### PR TITLE
Remove heron core fetching for MarathonScheduler.

### PR DESCRIPTION
**Pull request by cckellogg**
The heron core will be installed within the docker image.

### CODE REVIEW

**Comment by cckellogg** [comment]
@jrcrawfo what do you think of this approach?

**Comment by jrcrawfo** [comment]
In general, I don't think we should do this unless the Docker build process for adding those core binaries into the image are made available. Otherwise, it's difficult to test any changes to a new build since the Dockerfile is not available

**Comment by cckellogg** [comment]
What about adding a flag in the config on whether or not to download the heron core into the docker container?

**Comment by jrcrawfo** [comment]
I really like the idea of including the the core binaries in the image, but the Dockerfile you used to put the libraries in there should be made available to the user and built into the build process. Otherwise, it just seems very disconnected and when a new release is cut, there's no process for getting that new release built into a new docker image to be ran.

**Comment by srkukarni** [comment]
+1

**Comment by huijunw** [comment]
Will this PR be included in the 0.14.8?

**Comment by objmagic** [comment]
This is not for 0.14.8. And please fix merge conflict. Also someone please review it.

**Comment by kramasamy** [comment]
Ideally it will be nice to merge this - but we have some additional work to get this completed.

**Comment by cckellogg** [comment]
I added a pull request to compile and build a docker image for running heron. https://github.com/twitter/heron/pull/1934.

File: website/content/docs/operators/deployment/schedulers/dcos.md [hunk 1]
```diff
@@ -98,11 +99,8 @@ heron.marathon.scheduler.uri: "<core.dcos_url>/service/marathon"
 # Invoke the IScheduler as a library directly
 heron.scheduler.is.service:                  False
 
-# location of the core package
-heron.package.core.uri:                      https://github.com/twitter/heron/releases/download/0.14.7/heron-core-0.14.7-ubuntu.tar.gz
-
-# docker repo for executor
-heron.executor.docker.image: 'ndustrialio/heron-executor:jre8'
+# docker repo for heron with core packages installed
+heron.executor.docker.image: 'streamlio/heron:latest-ubunut14.04'
```
Comment (jrcrawfo) [inline website/content/docs/operators/deployment/schedulers/dcos.md]: With the tag, the only comment i have is to be sure we have tags corresponding to versions number. So, ideally, you'd be referencing something like `streamlio/heron:0.14.8`

**Comment by jrcrawfo** [comment]
Other than the Docker tagging comment above, which is something to be done outside of this anyway, this looks good 👍

File: website/content/docs/operators/deployment/schedulers/dcos.md (hunk 1 above)
Comment (cckellogg) [inline website/content/docs/operators/deployment/schedulers/dcos.md]: We are planning on publishing images for different operating systems so we came up with the convention heron:<heron-version>-<os-version>. So would publish a release image like this streamlio/heron:0.14.8-ubuntu14.04. What do you think?

**Pull request merged**